### ISSUE TITLE
aliexpress bootlegs

### ISSUE THREAD

**Issue by Alex12DEUT** [MiGeRA/FlashKit-MD-Plus#2]
hello

just to tell i have find a way how to reflash cheap and shit aliexpress bootlegs with a flashkit md device
need somes easy modifications to the bootlegs pcb

i can tell you how , but need add somes other who they use , s29gl serie work great and lile a real flashkit pcb , another toshiba one is a little bugy ( need erase it like a 29w serie , and program it like mx29lv320 , not alway work , need different command :)

**Comment by MiGeRA** [comment]
I'm glad that you use the result of my work and give feedback ;-)
What carts from aliexpress  do you try use with flashkit? (give links)
I have neither chips s29gl nor cartridges with their use for testing and possible updating of the sofware. However, judging by the documentation, the classic "bypass algorithm" (how for many 29-series), already implemented at the moment, is applicable for them...
Do you want to share your source code?

**Comment by Alex12DEUT** [comment]
All bootlegs of today and somes year ago use the same pcb like that one :
![6DA6C555-9DF0-410F-BDD0-DB40F0E4DB65](https://user-images.githubusercontent.com/122138317/218742291-677400b5-402a-4377-8e8a-b0bbf38f5825.jpeg)

just the flashchip change depend what they find in trash lol

I use any source code  just krikzz soft and your modded one  just need mod the pcb for make it flashkit compatible , need cut a line and add a wire , take 2 min to do , i would like the tuto shared to public when a majority if flashchip can be flashed :) but i can tell to you how do it if you have discord or something

**Comment by MiGeRA** [comment]
Carts with games offered for purchase as pre-programmed (not rewritable) - do not have the circuitry ability for rewrite them through the main-slot, as provided by the Flashkit interface developer. Modifying such a cartridge is easy: just find the WE-signal on the chip, disconnect it from "1" and connect it to pin B28 of the slot. (In rare cases it may be necessary to switch other circuits: WP, VPP and etc). Then the software does the work.
I also described a similar solution for modifying cartridges in the material on my website: https://migera.ru/smd/csp232.html and https://migera.ru/smd/dy2488.html
My software "Flashkit MD Plus" is the most complete and functional tools by control Flashkit hardware, alas (without claims and guarantees - but the result pleases me and my friends). Probably this topic is not in demand among interested people.
PS. If you may be communicate in Russian, then it would be easier for both of us ;-)
PPS. Do you propose to simply upgrade the description of the modified program? I'm sorry, but I may not be still can't grasp the essence of your thought ...

**Comment by Alex12DEUT** [comment]
Yes its like that need a wire from position of 5th pin right front ( CE ) to another bootleg pin and cut a line ( because vcc and CE are connected to each other on bootleg )
They seem to alway use the pcb i  show now so method is same for all for recent bootlegs

Maybe a generic flash code AAA/A9 , and AAA/AA  can be usefull for large compatibility ? I have it n python ( used for gba but same  ) I have another chip i will test today and sorry i am not russian lol

on the 3 types of flashchip i have 2 have been sucessfull reflashed ; one not possible

my idea is for increase the chip compatibility for user who want reflash bootleg  just that , i alrready have design my own real flashcard with m29w320 chip ^^ i have just do that for fun
